samples/rp/src/bin/epd7in5_v2.rs

```diff
@@ -0,0 +1,358 @@
+        );
+        Timer::after_millis(500).await;
+    }
+    Timer::after_secs(4).await;
+
+    // Display text with fast refresh mode
+    let mut epd = expect!(
+        epd.set_refresh_mode(&mut spi, RefreshMode::Fast).await,
+        "Failed to switch to fast refresh m"
+    );
+    info!("Displaying second text");
+    buffer
+        .fill_solid(&buffer.bounding_box(), BinaryColor::Off)
+        .unwrap();
+    let mut style = TextStyle::default();
+    style.alignment = Alignment::Center;
+    style.baseline = Baseline::Top;
+    let character_style = MonoTextStyle::new(&FONT_10X20, BinaryColor::On);
+    let text = Text::with_text_style(
+        "Hello, World!",
+        buffer.bounding_box().center(),
+        character_style,
+        style,
+    );
+    text.draw(&mut buffer).unwrap();
+    debug!(
+        "Text bounding box: ({},{}) ({},{})",
+        text.bounding_box().top_left.x,
+        text.bounding_box().top_left.y,
+        text.bounding_box().size.width,
+        text.bounding_box().size.height
+    );
+    expect!(
+        epd.display_framebuffer(&mut spi, &buffer).await,
+        "Failed to display text buffer"
+    );
+    Timer::after_secs(4).await;
+
+    // Display clock with partial refresh mode
+    let mut epd = expect!(
+        epd.set_refresh_mode(&mut spi, RefreshMode::Partial).await,
+        "Failed to set EPD to partial refresh mode"
+    );
+    buffer
+        .fill_solid(&buffer.bounding_box(), BinaryColor::Off)
+        .unwrap();
+    epd.write_base_framebuffer(&mut spi, &buffer).await.unwrap();
+    for time_str in ["12:24:31", "12:24:32", "12:24:33", "12:24:34", "12:24:35"] {
+        let mut style = TextStyle::default();
+        style.alignment = Alignment::Center;
+        style.baseline = Baseline::Top;
+        let character_style = MonoTextStyle::new(&FONT_10X20, BinaryColor::On);
+        let text = Text::with_text_style(time_str, Point::new(200, 300), character_style, style);
+        text.draw(&mut buffer).unwrap();
+        debug!(
+            "Text bounding box: ({},{}) ({},{})",
+            text.bounding_box().top_left.x,
+            text.bounding_box().top_left.y,
+            text.bounding_box().size.width,
+            text.bounding_box().size.height
+        );
+        expect!(
+            epd.display_partial_framebuffer(&mut spi, &buffer, text.bounding_box())
+                .await,
+            "Failed to display text buffer"
+        );
+        // Clear the text for the next update
+        buffer
+            .fill_solid(&text.bounding_box(), BinaryColor::Off)
+            .unwrap();
+        Timer::after_millis(1000).await;
+    }
+
+    let epd = expect!(epd.sleep(&mut spi).await, "Failed to put EPD to sleep");
+
+    Timer::after_secs(4).await;
+    let epd = expect!(epd.wake(&mut spi).await, "Failed to wake EPD");
+    let mut epd = expect!(
+        epd.init(&mut spi, RefreshMode::Fast).await,
+        "Failed to initialize EPD"
+    );
+
+    for i in 0..5 {
```

Comment:
It might be nice to have another `info!` before this section to explain what is being tested